contrib/lib/Cargo.toml

```diff
@@ -19,6 +19,7 @@ databases = ["r2d2", "rocket_contrib_codegen/database_attribute"]
 # User-facing features.
 default = ["json", "serve"]
 json = ["serde", "serde_json"]
+serde_ron = ["serde", "ron"]
```

Comment:
I'm not 100% sure about the naming here, but just 'ron' won't work because the crate already uses that name.

Reply:
I'm tempted to do a feature named `ron` and use `package=` to rename the crate locally. I want to check what other crates do before doing anything, though.

Reply:
Here's what I think: the feature cannot be named the same as the dependency, we should be consistent with the other features, and we can't rename those. Therefore I propose we use the feature `ron = ["serde", "ron_crate"]` along with the dependency `ron_crate = { version = "0.5.1", optional = true, package = "ron" }`. Code would refer to `ron_crate::whatever`.